jerry-core/ecma/base/ecma-helpers-conversion.c

```diff
@@ -324,6 +324,26 @@
 
 #define EPSILON 0.0000001
 
+#if CONFIG_ECMA_NUMBER_TYPE == CONFIG_ECMA_NUMBER_FLOAT64
+/**
+ * Number.MAX_VALUE and Number.MIN_VALUE value while using 64 bit float representation
+ * split into base and exponent part
+ */
+# define NUMBER_MAX_VALUE_BASE 1.7976931348623157
+# define NUMBER_MAX_VALUE_EXPONENT 308
+# define NUMBER_MIN_VALUE_BASE 5
+# define NUMBER_MIN_VALUE_EXPONENT -324
+#elif CONFIG_ECMA_NUMBER_TYPE == CONFIG_ECMA_NUMBER_FLOAT32
+/**
+ * Number.MAX_VALUE and Number.MIN_VALUE value while using 32 bit float representation
+ * split into base and exponent part
+ */
+# define NUMBER_MAX_VALUE_BASE 3.402823
+# define NUMBER_MAX_VALUE_EXPONENT 38
+# define NUMBER_MIN_VALUE_BASE 1.175494
+# define NUMBER_MIN_VALUE_EXPONENT -38
+#endif
```

Comment:
we used to add closing comments to `#endif /* ... */` (please existing number-related code for examples)